nginx/nginx.conf

```diff
@@ -37,8 +37,8 @@ http {
                                     application/vnd.ms-fontobject application/octet-stream
                                     application/x-font-woff multipart/related image/svg+xml;
 
-        ssl_certificate             /etc/nginx/private/cert.pem;
-        ssl_certificate_key         /etc/nginx/private/key.pem;
+        ssl_certificate             $SSL_CERT_FILE_PATH;
+        ssl_certificate_key         $SSL_KEY_FILE_PATH;
```

Comment:
I haven't yet tested this but don't you need to use `envsubst` to get this substitution to work?